shop/prestashop-performance/Dockerfile

```diff
@@ -1,7 +1,20 @@
-FROM prestashop/prestashop:1.7
+FROM prestashop/base:7.2-apache
+LABEL maintainer="devops team <devops@prestashop.com>"
+
+ENV PS_VERSION 1.7.7.0
+
+# Get PrestaShop
+ADD https://www.prestashop.com/download/old/prestashop_1.7.6.3.zip /tmp/prestashop.zip
```

Comment:
```suggestion
ADD https://www.prestashop.com/download/old/prestashop_1.7.6.9.zip /tmp/prestashop.zip
```

Reply:
Mmmmhh... Interesting.
I kept the 1.7.6.3 because it was the version we used during the benchmarks, so it could provide an easy way to reproduce our own tests.
But I agree it should evolve at one point.
It's up to you ;)